src/nfs/nfs_client_impl.cpp

```diff
@@ -212,7 +212,7 @@ void nfs_client_impl::end_get_file_size(::dsn::error_code err,
             _copy_requests_low.push(std::move(copy_requests));
     }
 
-    continue_copy();
+    tasking::enqueue(LPC_NFS_COPY_FILE, nullptr, [this]() { continue_copy(); }, 0);
```

Comment:
I'm uncertain if this is a good fix. I think `continue_copy` should not block at first.
As you pointed out, the blocking point is `folly::TokenBucket::consumeWithBorrowAndWait`.
We should use a non-blocking alternative `consumeWithBorrowNonBlocking` to prevent unnecessary block.
But anyway, we can leave it later.

Reply:
OK!